src/main/wrapper/ExecutionService.ts

```diff
@@ -140,6 +141,10 @@ export class ExecutionService {
                     const kicsResults = CxKicsRealTime.parseKicsRealTimeResponse(resultObject);
                     cxCommandOutput.payload = [kicsResults];
                     break;
+              case "cxLearnMoreDescriptions":
```

Comment:
The constant is defined with a capital C, will this work?

src/main/wrapper/CxConstants.ts
`LEARN_MORE_DESCRIPTIONS_TYPE = "CxLearnMoreDescriptions",`

Should we use the constant here instead of the direct string?